features/RST15_audit_events_admin_ui.feature

```diff
@@ -0,0 +1,95 @@
+Feature: Audit events convey useful information via admin interface
+
+    Background:
+	
+	Given fixtures from "bdd_auth" are loaded
+	And fixtures from "bdd_auth" are available
+        And fixtures from "auditevent_valid_excercise_ui" are loaded
+        And I am logged into the admin interface as "servicemanager"
+
+    Scenario: Audit entries can be listed
+
+	When I visit "/admin/plea/auditevent/"
+	Then I see a list of "audit event" items
+
+    Scenario: Audit entries can be sorted by URNs in "descending" order
+
+	When I visit "/admin/plea/auditevent/?o=1"
```

Comment:
I would do something like:
Given I am on the audit event page
When I sort "URN" by "descending" order
Then I see the "URN" results are sorted by "descending" order